dask/compatibility.py

```diff
@@ -12,4 +12,18 @@ def prod(iterable, *, start=1):
         return acc
 
 
-PY_VERSION = LooseVersion(".".join(map(str, sys.version_info[:3])))
+_PY_VERSION = LooseVersion(".".join(map(str, sys.version_info[:3])))
+
+
+def __getattr__(name):
+    if name == "PY_VERSION":
+        import warnings
+
+        warnings.warn(
```

Comment:
Wait, why not use the deprecated wrapper?

Reply:
Since `PY_VERSION` isn't a callable, I don't think a decorator will work in this situation (though I'm open to suggestions if you have any)

Reply:
ah true. I guess if we do this a lot we can write a helper function that takes variables rather than callables. But today isn't that day :)